mappings/net/minecraft/block/AbstractFluidBlock.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_2404 net/minecraft/block/AbstractFluidBlock
```

Comment:
yarn names dont use abstract from what i remember correct me if im wrong

Reply:
They do, u can do a quick github search on the yarn repo, will be presented with several examples.